tests/builder/custom_params_test.py

```diff
@@ -409,8 +409,9 @@ def test_ufo2ft_filter_roundtrip(self):
         self.assertEqual(self.ufo.lib[UFO2FT_FILTERS_KEY], ufo_filters)
```

Comment:
The test is good but looks misleading. Maybe you could add the comment that explains this:

The first half is testing the "goes 1 way" part of the diagram, which happens on instances (although the test is performed on a master, but the code works the same for both), and checks that the filter syntax is translated from Glyphs to ufo2ft.

The second half tests the roundtrip of ufo2ft filters from UFO to GSFontMaster and back to UFO.